evalit/mft/mft_automation.py

```diff
@@ -48,13 +50,15 @@ def run_automation(self):
         dest_storage_id = "d602acab-595a-44ac-a9b3-60c5ceca80e7"
 
         num_cores = multiprocessing.cpu_count()
-        transfer_ids = Parallel(n_jobs=num_cores)(delayed(self.submit_transfer)(file_name, source_storage_id, dest_storage_id) for file_name in self.file_list)
-
+        transfer_ids = Parallel(n_jobs=num_cores)(
+            delayed(self.submit_transfer)(file_name, source_storage_id, dest_storage_id)
+            for file_name in self.file_list
+        )
 
         start_time_map = {}
         end_time_map = {}
 
-        while(1):
+        while 1:
```

Comment:
👀

Reply:
We need to improve this. It's a legacy code. I don't like the infinite loop though.